gdc_ng_models/models/download_reports.py

```diff
@@ -22,6 +22,9 @@ class DataUsageReport(Base):
     report_period = db.Column(
         db.Date, primary_key=True, nullable=False)  # MM/YYYY 01/31/2019
 
+    all_report = db.Column(
```

Comment:
Do we also need to add this field to `to_json`?

Reply:
Shoot good point